meetings/bordeaux2018/index.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: common
+---
+
+# The Second Annual JuMP Developers Workshop
+
+June 28-29, 2018, at the Institut de Mathématiques de Bordeaux, University of Bordeaux
+
+## Purpose
+
+This workshop is a follow-up to the [first annual JuMP Developers Workshop](http://www.juliaopt.org/meetings/mit2017/) held in June 2017 at MIT. The purpose of the workshop is to bring together developers of mathematical optimization software based around the JuMP "stack". These include developers writing solvers in Julia, interfacing solvers from Julia, writing JuMP extensions (for problem classes such as stochastic, robust, and multiobjective), and developing other optimization libraries that utilize JuMP.
+
+The workshop is a great opportunity to meet other developers working on and around JuMP, as well as to provide feedback to the core JuMP team concerning its future direction.
+
+## Location
+
+The workshop will be held at the Institut de Mathématiques de Bordeaux, University of Bordeaux on the Thursday and Friday of the week prior to ISMP 2018 (i.e. June 28-29).
+
+A variety of information about Bordeaux, as well as how to get there, and possible accommodation options, is available on the [ISMP conference website](https://ismp2018.sciencesconf.org).
+
+## Call for participants
+
+The workshop is open for anybody to attend, however it is aimed at developers of mathematical optimization software who use JuMP rather than users. In particular, we encourage new contributors and those who have not met the core development team to attend.
+
+Participants are invited to give a 20 minute talk about their work that is related to JuMP. Talks should be aimed at a technical audience and demonstrate what you are using JuMP for, as well as the features that you like/dislike/have on your wishlist. Depending on the number of participants, we may not be able to accommodate all those who wish to talk. ([Here](https://www.youtube.com/watch?v=esOe5saQRKY&list=PLzK_rUGmc3o6EwPOCUCvBAbMJeYBS8PyY) is a playlist of talks from the previous workshop.)
+
+Please contact o.dowson at auckland.ac.nz if you would like to attend.
```

Comment:
Maybe `jump-dev-committee@googlegroups.com`? I've allowed moderated posting from non-members, and set you as a moderator :)